cengage/react-magma
# build: update package-lock.json using npm install --legacy-peer-deps

Issue: 
This pull request addresses the installation issue where dependencies needed to be resolved and npm ci

## What I did
I ran npm install --legacy-peers-deps to install the necessary dependencies. The only file that changed was package-lock.json.

## Screenshots:
N/A (no UI changes were made in this PR).

## Checklist 

## How to test
To verify this change:

- Clone the repository.
- Run npm ci --legacy-peer-deps.
- Ensure that the installation completes successfully and there are no changes for package-lock.json.
- Run unit tests to ensure all tests pass as expected.
